OpenRA.Game/InstalledMods.cs

```diff
@@ -72,23 +72,27 @@ Manifest LoadMod(string id, string path)
 					throw new InvalidDataException(path + " is not a valid mod package");
 
 				package = new Folder(path);
-				if (!package.Contains("mod.yaml"))
-					throw new InvalidDataException(path + " is not a valid mod package");
```

Comment:
Ah right, those service mods indeed don't need them.  You can make it ignore these errors for mods with Hidden: true.